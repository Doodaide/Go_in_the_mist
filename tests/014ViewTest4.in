who
place B18
view
place H18
view
place P16
view
place C17
view
place D2
view
place E3
view
place Q4
view
place R9
view
term