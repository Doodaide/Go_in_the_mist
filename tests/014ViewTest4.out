B
L18,xxxxxxxxxxxxxx...................................
G18,xxxxxxxxxxxxxx...........o.......................
C8,x......x......x......x......x......x......x......
B9,xx.....xx.....xx.....xx.....xx.....xx.....xx.....
B17,xxxxxxxxx.....xx.#...xx..o..xx.....xx.....xx.....
L1,............................xxxxxxxxxxxxxxxxxxxxx
S12,....xxx....xxx....xxx....xxx....xxx....xxx....xxx
G16,...........o.....................................
